templates/user/settings/keys_gpg.tmpl

```diff
@@ -47,7 +47,7 @@
 			<input type="hidden" name="title" value="none">
 			<div class="field {{if .Err_Content}}error{{end}}">
 				<label for="content">{{.i18n.Tr "settings.key_content"}}</label>
-				<textarea id="gpg-key-content" name="content" required>{{.content}}</textarea>
+				<textarea id="gpg-key-content" name="content" placeholder="Begins with '-----BEGIN PGP PUBLIC KEY BLOCK-----'" required>{{.content}}</textarea>
```

Comment:
This would need to be translatable.

```suggestion
				<textarea id="gpg-key-content" name="content" placeholder="{{.i18n.Tr "setting.key_content_gpg_placeholder}}" required>{{.content}}</textarea>
```

With an associated addition to locale_en-US.ini

Reply:
I don't think there are localized version of these fences?

Reply:
that's what I meant by: "With an associated addition to locale_en-US.ini" ...

Reply:
Right, the `Begins with` part can be localized.

Reply:
ok, addressed